languages/llvmir/README.md

```diff
@@ -0,0 +1,28 @@
+# JPlag LLVM IR language module
+
+The JPlag LLVM IR module allows the use of JPlag with submissions in the LLVM IR. <br>
```

Comment:
The `<br>` is responsible for the line-break. Without it the text would continue. I added it to stay consistent because it is in the Kotlin and CPP2 README too.